### PR TITLE
Document the “plugins” property of command steps

### LINKED ISSUE #275: Buildkite plugin step
Would it be possible to get some more docs around **https://buildkite.com/docs/agent/v3/plugins**, please?

E.g. can standalone plugin-steps target specific agents?

### PR DESCRIPTION
**Pull request by joscha**
See https://github.com/buildkite/docs/issues/275#issuecomment-400604220

closes #275 

cc @lox

### CODE REVIEW

**Comment by toolmantim** [comment]
Thanks so much for the contribution @joscha! We should also mention the `plugin` property in the list of [Command Step Attributes](https://buildkite.com/docs/pipelines/command-step#command-step-attributes) as well. I'll get this fixed up and merged in for you.

File: pages/pipelines/plugins.md.erb [hunk 1]
```diff
@@ -3,73 +3,74 @@
...
+  - command: yarn install && yarn run test
+    plugins:
+      docker#v1.0.0:
+        image: node
+        workdir: /app
+```
 
-A real-life example is the [Docker Compose plugin]. It provides an integration
-with docker-compose that assists with running your builds within a docker environment:
+More advanced plugins, such as [Docker Compose plugin], can be run multiple times within a build, and use [buildkite-agent meta-data] to share data between the steps:
```
Comment (harrietgrace) [inline pages/pipelines/plugins.md.erb]: What does 'can be run multiple times in a build' mean? I think we need to explain a bit earlier (before the docker plugin example too), what it means to 'run' a plugin. In the first line we say that the hooks are 'downloaded and run as part of your builds', maybe that's the point where we could be clearer? 

I don't think this comment is very clear either 😅I'm mostly confused by the bits that say that the plugins are hooks, but they're 'run' like a script, but also that there's configuration too and I'm not sure where that comes in/how it's executed/what it's for?

Comment (toolmantim) [reply pages/pipelines/plugins.md.erb]: Thanks, they're good questions! I've tried to clarify how that works in the latest commit…

**Comment by toolmantim** [comment]
@harrietgrace thanks! Yeah, usually how it works is that every fresh look at the doc I'll be "Oh darn, so much to improve" — I reckon the whole config part is really key, and totally missing.

**Pull request merged**